sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Queues/QueuesOptions.cs

```diff
@@ -140,14 +140,15 @@ public TimeSpan MaxPollingInterval
         }
 
         /// <summary>
-        /// Gets or sets the number of times to try processing a message before moving it to the poison queue (where
-        /// possible).
+        /// Gets or sets the number of times to try processing a given blob before adding a message to a
+        /// storage queue named, webjobs-blobtrigger-poison.
+        ///
+        /// If not specified, will default to 5.
+        ///
+        /// See <see href="https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob-trigger#poison-blobs">
+        /// Poison Blobs
+        /// </see>.
```

Comment:
If I understood @mathewc's comment correctly, this can remain the same - it's the XML doc for BlobOptions.MaxDequeueCount that should be updated?

Reply:
Yes right. I missed that this was an update to QueuesOptions not BlobOptions where it should be

Reply:
ah I definitely just updated the wrong docs. Will update now.